dexbot/cli_conf.py

```diff
@@ -194,7 +196,9 @@ def configure_dexbot(config, ctx):
                 break
         setup_systemd(whiptail, config)
     else:
-        bitshares_instance = ctx.bitshares
+        bitshares_instance = BitShares(
```

Comment:
The wallet is locked in a new instance of BitShares, the unlock decorator was there to allow canceling orders when editing/removing workers.